vars/main.yml

```diff
@@ -3,9 +3,23 @@
 ---
 # vars file for telegraf-docker-in-docker
 
+# Link: https://github.com/influxdata/influxdata-docker/blob/master/telegraf/1.35/alpine/Dockerfile
+# TODO Add armv8 once the correct string is known
+tdid_arch_has_alpine_image: >-
+  {{
+    (ansible_architecture == 'x86_64')
+  }}
+
+# Link: https://github.com/influxdata/influxdata-docker/blob/master/telegraf/1.35/Dockerfile
 # TODO Add armv8 once the correct string is known
 tdid_arch_is_supported: >-
   {{
     (ansible_architecture == 'x86_64') or
     (ansible_architecture == 'armv7l')
   }}
+
+tdid_docker_image_is_supported: >-
+  {{
+    tdid_arch_is_supported and
+    not (tdid_docker_image is search("alpine") and not tdid_arch_has_alpine_image)
```

Comment:
It may be beneficial to explain the last line in a short comment. Compared to the rest of the file it is rather hard to read.

Reply:
Added another commit with an explaining comment.

Reply:
Okay, I found it not so obvious. 😅 Maybe another sentence in the documentation on the user visible variable would be good. The detection logic itself is rather just an implementation detail.